packages/internal/src/module-utils.js

```diff
@@ -25,3 +26,18 @@ export const getSpecifier = async fileUrl => {
   return `${name}/${moduleSpecifier.replace(/^\.\//, '')}`;
 };
 harden(getSpecifier);
+
+/**
+ * Given an import specifier and importer URL (usually import.meta.url), return
+ * a file system path corresponding with the specifier for the importer.
+ *
+ * @param {string} specifier
+ * @param {string} baseURL
+ * @returns {string}
+ */
+export const resolveToPath = (specifier, baseURL) => {
```

Comment:
I wonder whether this carries its weight. `createRequire(baseURL).resolve(specifier)` is just about as concise as importing this.

Reply:
I had the same thought, and decided that it was better than remembering `new URL(…).pathname` (or rather, `fileURLToPath(new URL(…))` per #10082). But I agree with you that `createRequire` is better where available, and have switched inquisitor.test.ts to that. I don't know whether or not it makes sense to leave this helper function for a followup to fix existing uses of `import-meta-resolve`, but have opted to do so for now.